src/common/paths/users_userID_password.yml

```diff
@@ -18,6 +18,82 @@ post:
 
     #### Related guides
 
+    - [InfluxDB Cloud - Change your password](https://docs.influxdata.com/influxdb/cloud/account-management/change-password/)
+    - [InfluxDB OSS - Change your password](https://docs.influxdata.com/influxdb/latest/users/change-password/)
+  parameters:
+    - $ref: '../parameters/TraceSpan.yml'
+    - in: path
+      name: userID
+      schema:
+        type: string
+      required: true
+      description: The ID of the user to set the password for.
+  requestBody:
+    description: The new password to set for the user.
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '../schemas/PasswordResetBody.yml'
+  responses:
+    '204':
+      description: Success. The password is updated.
+    '400':
+      description: |
+        Bad request.
+
+        #### InfluxDB Cloud
+
+        - Doesn't allow you to manage passwords through the API; always responds with this status.
+
+        #### InfluxDB OSS
+
+        - Doesn't understand a value passed in the request.
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/Error.yml"
+          examples:
+            updatePasswordNotAllowed:
+               summary: Cloud API can't update passwords
+               value: {
+                        "code": "invalid",
+                        "message": "passwords cannot be changed through the InfluxDB Cloud API"
+                      }
+    default:
+      description: Unexpected error
+      $ref: '../responses/ServerError.yml'
+  x-codeSamples:
+    - lang: Shell
+      label: "cURL: use Basic auth to update the user password"
+      source: |
+        curl --request POST \
+           "http://localhost:8086/api/v2/users/USER_ID/password" \
+          --header 'Content-type: application/json' \
+          --user "USERNAME:PASSWORD" \
+          --data-binary @- << EOF
+            {"password": ""}
+        EOF
```

Comment:
I don't know that this is correct. Unless I'm missing something, this API only supports token or session auth so this curl will always just bounce back as unauthed.

Reply:
> I don't know that this is correct. Unless I'm missing something, this API only supports token or session auth so this curl will always just bounce back as unauthed.

True, doesn't seem to work with Basic Auth.

Reply:
cURL example with session would be something like:

```sh
  curl -v -c ./cookie-file.tmp --request POST \
   	"$INFLUX_URL/api/v2/signin" \
    --user "${INFLUX_USER_NAME}:${INFLUX_USER_PASSWORD}"


  curl -v --request POST \
    -b ./cookie-file.tmp \
"$INFLUX_URL/api/v2/users/$INFLUX_USER_ID/password" \
    --header 'Content-type: application/json' \
    --data-binary @- << EOF
      {"password": "NEW_USER_PASSWORD"}
EOF
}
```

Reply:
The `POST` example is updated to:

```console
curl --request POST \
           "http://localhost:8086/api/v2/users/USER_ID/password" \
          --header 'Content-type: application/json' \
          --header "Authorization: Token INFLUX_TOKEN" \
          --data-binary @- << EOF
            {"password": "NEW_USER_PASSWORD"}
        EOF
```

and `PUT` example is updated to:

```console
curl -c ./cookie-file.tmp --request POST \
   "http://localhost:8086/api/v2/signin" \
  --user "${INFLUX_USER_NAME}:${INFLUX_USER_PASSWORD}"

curl -b ./cookie-file.tmp --request PUT \
   "http://localhost:8086/api/v2/users/USER_ID/password" \
  --header 'Content-type: application/json' \
  --data-binary @- << EOF
    {"password": "NEW_USER_PASSWORD"}
EOF
```